okhttp/src/main/java/okhttp3/Call.java

```diff
@@ -57,6 +57,8 @@
    */
   Response execute() throws IOException;
 
+  Response execute(DiskCacheListener diskCacheListener) throws IOException;
```

Comment:
This API change feels way to "in your face", this is the key part of the okhttp client experience, and will now prompt users to choose between two options for what I presume is a niche feature.  Not all users use caches, and not all those will want a listener.